src/test/java/com/google/api/codegen/testdata/csharp/csharp_library.baseline

```diff
@@ -5218,28 +5235,28 @@ namespace Google.Example.Library.V1.Tests
 
 // Generated code. DO NOT EDIT!
 
-using Google.Api.Gax;
-using Google.Api.Gax.Grpc;
+using gax = Google.Api.Gax;
+using gaxgrpc = Google.Api.Gax.Grpc;
 using Google.Cloud.Tagger.V1;
-using Google.LongRunning;
-using Google.Protobuf;
-using Google.Protobuf.WellKnownTypes;
+using lro = Google.LongRunning;
+using proto = Google.Protobuf;
+using protowkt = Google.Protobuf.WellKnownTypes;
 using Google.Tagger.V1;
-using Grpc.Core;
-using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
+using grpccore = Grpc.Core;
+using s = System;
+using sc = System.Collections;
+using scg = System.Collections.Generic;
+using sco = System.Collections.ObjectModel;
 using System.Linq;
```

Comment:
It feels odd for this to be unqualified. Any reasons?

Reply:
Unqualified because of extension method use. I'd rather leave this as it is for now. If it causes problems, then it can be changed in a future PR.